docs/modules/ROOT/pages/how-to/use-k8up.adoc

```diff
@@ -0,0 +1,373 @@
+--
+<1> This `Secret` contains the credentials used to store data in the MinIO instance.
+    *These credentials are only for demo purposes!*
+<2> Remember to customize the parts marked as `[YOUR_USERNAME]` and `[YOUR_CHOSEN_ZONE]` to your liking (and according to the xref:references/zones.adoc[Zones documentation page]).
+
+include::partial$url-length-warning.adoc[]
+
+Apply the deployment to your {product} project:
+
+[source,shell]
+--
+oc apply -f minio.yaml
+--
+
+After a few minutes you should be able to open the MinIO GUI console at the following URL:
+
+`https://[YOUR_USERNAME]-minio-gui.apps.[YOUR_CHOSEN_ZONE].appuio.cloud`
+
+Login using the `minio` username and the `minio123` password, as specified in the deployment above.
+
+=== 4. Apply a Backup object
+
+Add a file named `backup.yaml` at the root of the project with the following information:
+
+[source,yaml]
+--
+apiVersion: v1
+kind: Secret # <1>
+metadata:
+  name: backup-repo
+type: Opaque
+stringData:
+  password: p@ssw0rd
+---
+apiVersion: k8up.io/v1
+kind: Backup
+metadata:
+  name: backup-test
+spec:
+  failedJobsHistoryLimit: 2
+  successfulJobsHistoryLimit: 2
+  backend:
+    repoPasswordSecretRef:
+      name: backup-repo
+      key: password
+    s3:
+      endpoint: http://minio:9000
+      bucket: backups
+      accessKeyIDSecretRef:
+        name: minio-credentials
+        key: username
+      secretAccessKeySecretRef:
+        name: minio-credentials
+        key: password
+--
+<1> This `Secret` is used to store the password used by Restic to access the backup repository stored in MinIO.
+    *This credential is only for demo purposes!*
+
+Apply the manifest:
+
+[source,shell]
+--
+oc apply -f backup.yaml
+--
+
+A few seconds after applying this `Backup` object, you will see in the MinIO GUI that the Restic repository has been successfully initialized and populated with a bucket called `backups`.
+
+=== 5. Restore the backup locally
+
+Using the `restic` command line tool, we will now restore the backup locally on your system.
+Create a script named `environment.sh` that exports the required environment variables used by https://restic.net/[restic]:
+
+[source,bash]
+--
+#!/usr/bin/env bash
+
+# <1>
+export RESTIC_REPOSITORY=s3:http://[YOUR_USERNAME]-minio-server.apps.[YOUR_CHOSEN_ZONE].appuio.cloud/backups/
+export RESTIC_PASSWORD=p@ssw0rd       # <2>
+export AWS_ACCESS_KEY_ID=minio        # <3>
+export AWS_SECRET_ACCESS_KEY=minio123 # <3>
+--
+<1> Remember to change the `[YOUR_CHOSEN_ZONE]` and `[YOUR_USERNAME]` placeholders to your preferred {product} zone and your username, respectively.
+<2> This is the Restic repository backup password.
+<3> These are the MinIO credentials.
+
+Source the file to load the variables in your environment:
+
+[source,shell]
+--
+source environment.sh
+--
+
+Use `restic` to watch the current snapshots:
+
+[source,shell]
+--
+restic snapshots
+--
+
+You can now backup the latest snapshot locally into the `restored` folder:
```

Comment:
Maybe an example with a restore directly to a PVC would make more sense?

Reply:
Well, there's the K8up website and its getting started guide for that. Just wanted to show that effectively, stuff is backed up.